api/Document.json

```diff
@@ -7671,14 +7681,10 @@
               "notes": "Doesn't fire the <code>visibilitychange</code> event when the browser window is minimized, nor when <code>hidden</code> is set to true."
             },
             "safari": {
-              "version_added": "7",
```

Comment:
Why is the old entry being removed? I guess it's wrong, that `onvisibilitychange` wasn't exposed before Safari 10.1, but perhaps the "visibilitychange" event was still fired?

If sorting this out will also involve updating the `visibilitychange_event` entry, can you break the change out of this PR?

Reply:
Yeah, this is a similar situation to #6833.  I can split this out and add the note like the other PR?

Reply:
Yes, that would be great! It's the only remaining blocker for this PR to land.

Reply:
Cherry-picked as #6849!

Reply:
Can you also revert these changes?

Reply:
I've reverted it now.